pylint/lint/__init__.py

```diff
@@ -41,7 +37,6 @@
     "ArgumentPreprocessingError",
     "_patch_sys_path",
     "fix_import_path",
-    "preprocess_options",
 ]
```

Comment:
Or should I add it back? Why is this in the public API?

Reply:
It's because ``pylint/lint.py`` was a gigantic file that we burst into a module so everything that was in it originally became an API in order to not make any assumption and break existing code.

Reply:
Do you want me to add it back? I did change the signature... So we would need to do something with `DeprecationWarnings`.

Reply:
It wasn't clear from the above message but I think we can start to make some assumptions in order to not have to maintain useless API forever.

Reply:
> we would need to do something with DeprecationWarnings.

Yeah no, I think it's something that is really pylint internal. I don't think it's used. It could but I don't see how.